docs/code/modules/index.md

```diff
@@ -508,7 +508,7 @@ AlgoNode (mainnet)
 
```

Comment:
This change was hanging about in my local and is unrelated to the rest, but harmless to include